handlers/metrics.go

```diff
@@ -33,7 +33,7 @@ var (
 			Name: "router_backend_handler_response_duration_seconds",
 			Help: "Histogram of response durations by router backend handlers",
 			Buckets: prometheus.ExponentialBuckets(
-				0.25, 2, 6, // This buckets [...0.25  0.5  1  2  4  8...]
+				0.0625, 2, 7, // This buckets [...0.0625  0.125  0.25  0.5  1  2  4...]
 			),
```

Comment:
63ms still seems a bit high to me as a lower bound. I'd recommend just reverting to the [default buckets](https://github.com/prometheus/client_golang/blob/master/prometheus/histogram.go#L63), which will give a much better range and similar accuracy. i.e. just delete lines 35-37.

Reply:
Ah yeah, sounds reasonable. I'll do that.

I guess it increases the number of labels (do the cool kids call that cardinality?) a bit - the default buckets have ~12, whereas we only had ~6. But it's a useful metric, so I guess we can try it and see.

Reply:
Ah yeah, thanks for that! Addressed it in 113bb28

Reply:
Yeah a few extra buckets should be absolutely fine here - the cardinality (of the sets of possible or in-use combinations of label values) only becomes an issue if labels are being misused or if you start specifying hundreds of buckets (either of which tend to cause problems in their own right). Thanks for fixing!